.github/workflows/ci.yml

```diff
@@ -34,52 +33,91 @@ jobs:
           pip install wheel
           curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
 
+      - name: Setup cache variables
+        shell: bash
+        id: cache-vars
+        run: |
+          export PATH="$HOME/.poetry/bin:$PATH"
+          echo "::set-output name=pip-dir::$(pip cache dir)"
+          echo "::set-output name=poetry-venv-dir::$(poetry config virtualenvs.path)"
+          echo "::set-output name=date::$(/bin/date -u "+%m%w%Y")"
+
+      - name: Setup Pip cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.cache-vars.outputs.pip-dir }}
+          key: ${{ runner.os }}-pip-${{ env.pythonLocation }}-${{ hashFiles('poetry.lock') }}
```

Comment:
since we are caching venv this seems unnecessary. thought?

Reply:
Yeah, poetry venv should contain the dependencies.